### PR TITLE
/tests/rbd/rbd_system.py

### PR DESCRIPTION
**Pull request by harishmunjulur**
Signed-off-by: harishmunjulur <hmunjulu@redhat.com>

### CODE REVIEW

**Comment by harishmunjulur** [comment]
3.3-rhel-7: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1595875970496/

4.1-rhel-8: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1595871880102/

File: tests/rbd/rbd_system.py [hunk 1]
```diff
@@ -15,7 +13,9 @@ def run(**kw):
     client_node = rgw_client_nodes[0]
     # cleanup any existing stale test dir
     test_folder = 'rbd-tests'
-    client_node.exec_command(cmd='rm -rf ' + test_folder)
+    client_node.exec_command(cmd='sudo yum -y install python3')
+    client_node.exec_command(cmd='python3 -m venv ' + test_folder)
+    client_node.exec_command(cmd='sudo rm -rf ' + test_folder)
```
Comment (rakeshgm) [inline tests/rbd/rbd_system.py:18]: both these statements are contradictory. you are creating virtualenv with rbd-tests and deleting the folder in the next line. And you are still not activating virtualenv, you need to activate it and then install the packages. right now the packages are getting installed in system installed version of python. can you correct these

**Comment by rakeshgm** [comment]
@harishmunjulur there is one more PR for this. can you close one of them. https://github.com/red-hat-storage/cephci/pull/143

**Comment by harishmunjulur** [comment]
4.1-rhel-8 - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1596076455101/
3.3-rhel-7 - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1596065439274/

**Comment by rakeshgm** [comment]
@harishmunjulur travis-ci is failing.

**Comment by XDsubhash** [comment]
@harishmunjulur please rosolve travisci checks
run flake8 before sending pr 

py37 run-test: commands[1] | flake8
./tests/rbd/rbd_system.py:31:121: E501 line too long (188 > 120 characters)
./tests/rbd/rbd_system.py:31:158: E225 missing whitespace around operator
./tests/rbd/rbd_system.py:31:175: E225 missing whitespace around operator

**Comment by harishmunjulur** [comment]
4.1-rhel-8 - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1596562758617/
4.1-rhel-7 - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1596570714895/
3.3-rhel-7 - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1596556545068/

**Comment by rakeshgm** [comment]
looks good.

**Pull request merged**